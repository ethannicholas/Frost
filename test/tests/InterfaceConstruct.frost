interface Fail {
}

method main() {
    def f := Fail()
}
