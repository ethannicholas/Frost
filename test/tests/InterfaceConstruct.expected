InterfaceConstruct.frost:5:14: error: 'anonymous.Fail' is an interface and may not be directly constructed
1 error
